Update user_desktop_wallet_guide.rst (#33)
BeamMW/beam-docs: source/rtd_pages/user_desktop_wallet_guide.rst

.. figure:: images/desktop/01_downloading/02.png
   :alt: Downloads page

> Ensure that you’ve downloaded the untampered Wallet installation file by checking that the value generated by SHA256 is the same as announced on the Beam official site:

Reviewer: We shouldn't delete `.. note::` since it important part for us.
But here there's typo missed space after `.. note::`
```suggestion
.. note:: Ensure that you’ve downloaded the untampered Wallet installation file by checking that the value generated by SHA256 is the same as announced on the Beam official site:
```

   :alt: Mac Install

> When you try to open the wallet for the first time, you will receive a security warning stating that Beam Wallet was created by an unrecognized developer. We are working to eliminate the warning; meanwhile follow the steps below to launch the Beam Wallet app on your Mac.

Reviewer: Let's replace:
`warning; meanwhile follow`
to
`warning; meanwhile, follow`
```suggestion
When you try to open the wallet for the first time, you will receive a security warning stating that Beam Wallet was created by an unrecognized developer. We are working to eliminate the warning; meanwhile, follow the steps below to launch the Beam Wallet app on your Mac.
```

Did you write down your seed phrase correctly? Triple-check your handwriting again. The difference between *_F_unnel* and *_T_unnel* can be crucial when trying to restore a wallet with valuable funds in the far or near future.

Did you verify your handwriting? Now go find a safe place for the piece of paper!

.. important:: Storing the seed phrase on your computer makes your funds prone to cyber attacks (read: much *less* secure). 'Creative' approaches like saving a screenshot of the wallet or your handwriting on your computer or in the cloud *may* sound like a good idea, but it is absolutely **not recommended.** If hackers get access to your computer, network drive, etc., they can can potentially steal your seed phrase by using OCR programs (which can scan pictures and transform them into plain text) and thereby get access to your funds.

.. figure:: images/desktop/03_creating_new_wallet/04.png
   :alt: Keeping seed phrase safe warning

.. attention:: Always store your seed phrase in a safe and secure location (and better more than one in different geo locations). Write it on a piece of paper. Do not store it electronically, either as plain text or in any other form!

> In order to ensure that you have really written down your seed phrase, you will be asked to fill in several words from your seed phrase in a random order.

Reviewer: `in a random`
here grammatically `a` can be missed
replace it back to
`in random`
```suggestion
In order to ensure that you have really written down your seed phrase, you will be asked to fill in several words from your seed phrase in random order.
```

.. figure:: images/desktop/03_creating_new_wallet/05.png
   :alt: Repeat your seed phrase

> Only when you typed all the requested words correctly, will you be able to proceed to the next step.

Reviewer: `will you be able to proceed to the next step.`
It's not a question, need to change word order but I agree to replace `allowed` to `be able`
Let's replace it to:
`you will be able to proceed to the next step`
```suggestion
Only when you typed all the requested words correctly, you will be able to proceed to the next step.
```

If, during installation, the wallet discovers one or more previous versions installed, it will propose to migrate to the new version

.. figure:: images/desktop/upgrade/01.png
   :alt: Start migration

If you do not wish to migrate an existing wallet, you can also create a new wallet or choose the location of the wallet database file yourself by clicking ‘Restore wallet or create a new one’.

.. note:: Migration will make a copy of the existing wallet database (wallet.db), keeping all the data intact.

If you choose ‘start auto migration’, on the next screen, you will be asked to choose which wallet database file you want to migrate. To choose manually, pick the ‘Select wallet database file manually’ option.

.. figure:: images/desktop/upgrade/02.png
   :alt: Select the wallet database file

Once the file is selected, click 'next' to enter the database password.

.. figure:: images/desktop/upgrade/03.png
   :alt: Starting the migration

Once you enter the correct password, the migration will complete, and the wallet will resume synchronization with the blockchain.

.. figure:: images/desktop/upgrade/04.png
   :alt: Migration

Main screen
-----------

> Once your wallet is created, the main screen will show. The main screen of the wallet shows the current balance in the Amount status field, as well as the transaction history and statuses. There are several transactions tabs: All, In progress, Sent, and Received. To the right of these tabs, there are a search bar and export-csv-file and payment-proof-verification buttons. On the left, under the Beam logo, there is a toolbar that provides navigation between different wallet screens, such as `Main`, `Atomic Swaps`, `Addresses`, `Notifications`, `UTXO`, and `Settings`.

Reviewer: Let's replace :
- `Once your wallet is created, the main screen will show.` to `Once your wallet is created, the main screen will be shown`.
- `are a search bar and export-csv-file and payment-proof-verification buttons.` to `are a search bar, export-csv-file, and payment-proof-verification buttons.`
```suggestion
Once your wallet is created, the main screen will be shown. The main screen of the wallet shows the current balance in the Amount status field, as well as the transaction history and statuses. There are several transactions tabs: All, In progress, Sent, and Received. To the right of these tabs, there are a search bar, export-csv-file, and payment-proof-verification buttons. On the left, under the Beam logo, there is a toolbar that provides navigation between different wallet screens, such as `Main`, `Atomic Swaps`, `Addresses`, `Notifications`, `UTXO`, and `Settings`.
```

.. figure:: images/desktop/04_main_screen/01.png
   :alt: Main screen

Wallet status indicator
-----------------------

In the top left corner of the Main Screen, under 'Wallet', you see the connection status indicator.

> A green indicator means that the node is up and running, connected to peers.

Reviewer: Let's replace :
- `A green` to `The green` 
```suggestion
The green indicator means that the node is up and running, connected to peers.
```

.. figure:: images/desktop/04_main_screen/02.jpg
   :alt: Main screen

.. note:: For an integrated node, the indicator has a lock shape.

.. figure:: images/desktop/04_main_screen/04.png
   :alt: Lock shape indicator
   
Red means it is either unable to connect to the node, or the node has a problem you can help resolve.

.. figure:: images/desktop/04_main_screen/03.jpg
   :alt: Wallet disconnected

In brackets, the indicator also specifies the network to which the wallet is connected:

* **Mainnet:** the network with real money and actual transactions.
* **Testnet:** staging environments for trying new features in our wallet, node, and miner software. For advanced users only.

> * **Masternet:** new features under development; if you see this in your wallet, it means you are a very early adopter or Beam code contributor; otherwise you’ve probably arrived atnext to the wrong place.

Reviewer: Looks like a typo: `arrived atnext to`
Replace it to `arrived next to`
```suggestion
* **Masternet:** new features under development; if you see this in your wallet, it means you are a very early adopter or Beam code contributor; otherwise you’ve probably arrived next to the wrong place.
```

Sending BEAM
------------

Before starting to send BEAM, please first read about what an :ref:`address` is.

Here is what the process of sending BEAM looks like from a Sender’s perspective:

* Receive the address the funds should be sent to
* Send BEAM to the Receiver
* Stay online until the Receiver confirms the transaction

Receiving the address
---------------------

.. attention:: Make sure that the address is received untampered with, by using a **secure communication channel**.

.. attention:: When copying the address to the Beam Wallet app, please verify visually that the address in the wallet looks exactly like the address in the secure messaging app. This is necessary because viruses and malware on your computer may change an address while it’s in the clipboard.

Sending funds
-------------

> In order to send BEAM, please click the magenta ‘Send’ button in the top right corner. This will open the Send screen.

Reviewer: ```suggestion
In order to send BEAM, you need to click the magenta ‘Send’ button in the top right corner. This will open the Send screen.
```

.. figure:: images/desktop/06_sending_beam/01.png
   :alt: Send BEAM screen

Make sure you have the correct address and paste the Receiver’s Beam address into the 'Send To' field.

To help identify the transaction, you may also choose to fill in the optional Comment field. The comment will remind you what or who the transaction is for. The comment is stored locally; thus it will only be visible in your wallet for bookkeeping purposes.

The comment can be seen on the `Address screen`_:

.. figure:: images/desktop/06_sending_beam/02.png
   :alt: Send BEAM screen

The comment is also displayed in the extended transaction view on the Main screen:

Select the transaction amount, in BEAM, you want to send. The transaction amount is in BEAM and may contain fractional values, such as 1.25 BEAM or 11.3 BEAM and the like. Keep in mind you also have to pay a transaction fee; hence the amount to send plus the fee cannot exceed your available balance.

.. figure:: images/desktop/06_sending_beam/04.png
   :alt: Send BEAM amount

> Transaction fees are specified in GROTH (100 millionths of a BEAM).The minimum fee is 100 GROTH; it’s set by default, but a higher transaction fee will motivate miners to prioritize your transaction. 

Reviewer: ```suggestion
Transaction fees are specified in GROTH (100 millionths of a BEAM). The minimum fee is 100 GROTH; it’s set by default, but a higher transaction fee will motivate miners to prioritize your transaction. 
```

*Example:* There are three wallets: One belongs to you, one belongs to Alice, and another belongs to Bob. You have created one address to receive money from Alice and another address to receive money from Bob. Alice and Bob will never know that they are sending money to the same person.

The same address can be used for sending and receiving money.

*Example:* you have created an address to send money to Alice. Alice can see the address the money came from and can send money to that address, back to you.

.. attention:: For ultimate privacy, it is advised to have a **dedicated address for every transaction,** whether sending or receiving.

.. attention:: Although not recommended, an address can be reused until it had reached its expiration time (24 hours from when it was generated).

*Example:* Imagine you’ve created an address with an expiration interval of 24 hours and immediately sent it to Alice. In the next 24 hours, Alice will be able to send BEAM to you as many times as she likes, reusing the same address of yours.

> .. warning:: Reusing same addresses, created with a longer expiration interval, can be convenient, but the tradeoff of decreased privacy should be kept in mind.

Reviewer: ```suggestion
.. warning:: Reusing the same addresses, created with a longer expiration interval, can be convenient, but the tradeoff of decreased privacy should be kept in mind.
```

.. figure:: images/desktop/10_settings_screen/01.png
   :alt: Setting screen

.. note:: At the top right corner of the screen, the version is displayed. Whenever you ask for support or report issues, it is important to specify the version you're using.

> The ‘General settings’ section allows you to get access to external links, setup additional display currencies for amounts and the screen-lock time. The Wallet will automatically lock after the time you choose here, to protect your funds from unauthorized access. Also in this section, you can see the current location of the wallet files.

Reviewer: ```suggestion
The ‘General settings’ section allows you to get access to external links, setup additional display currencies for amounts, and screen-lock time. The Wallet will automatically lock after the time you choose here, to protect your funds from unauthorized access. Also in this section, you can see the current location of the wallet files.
```

Proof of transaction
--------------------

> Starting from version 2.0, the receiver's wallet automatically signs proofs of received transaction and sends them to the senders. This proof of transaction will be added to any outgoing transaction that was successfully completed (transaction's status is 'sent'). Proof of transaction is not available for transaction with a different status ('expired', 'failed', 'completed', or 'received'). Upon request, the sender can provide a proof of the transaction by following the procedure below:

Reviewer: ```suggestion
Starting from version 2.0, the receiver's wallet automatically signs proofs of the received transaction and sends them to the senders. This proof of transaction will be added to any outgoing transaction that was successfully completed (transaction's status is 'sent'). Proof of transaction is not available for transactions with a different status ('expired', 'failed', 'completed', or 'received'). Upon request, the sender can provide proof of the transaction by following the procedure below:
```

.. figure:: images/desktop/rescan/03.png
   :alt: Click on the rescan button

3. Verify that your balance, transactions, and UTXO states are up to date.

> .. warning:: In most cases the wallet rescans immediately, so you can check you balance right after you’ve pressed the ‘rescan’ button. The rescan speed depends on UTXO value. 

Reviewer: ```suggestion
.. warning:: In most cases, the wallet rescans immediately, so you can check your balance right after you’ve pressed the ‘rescan’ button. The rescan speed depends on the UTXO value. 
```
